src/common/exceptions/user-already-exists-exception.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { CustomErrorCodes } from '@common/customErrorCodes/custom-error-codes';
```

Comment:
```suggestion
import { CustomErrorCodes } from '@common/enums';
```
should rename folder from `customErrorCodes` to `enums`
should add `index.ts `
should rename as `custom-error-code.enum.ts`